Libraries/Image/RCTImageLoader.m

```diff
@@ -648,16 +644,40 @@ - (RCTImageLoaderCancellationBlock)getImageSize:(NSString *)imageTag
     return;
   }
 
-  [_bridge.imageLoader loadImageWithTag:uri callback:^(NSError *error, UIImage *image) {
+  [self loadImageWithTag:uri callback:^(NSError *error, UIImage *image) {
     if (error) {
       reject(RCTErrorPrefetchFailure, nil, error);
       return;
     }
 
-    resolve(@YES);
+    resolve([self getMetadataForImage:image]);
   }];
 }
 
+RCT_EXPORT_METHOD(getMetadata:(NSString *)uri
+                      resolve:(RCTPromiseResolveBlock)resolve
+                       reject:(RCTPromiseRejectBlock)reject)
+{
+  if (!uri.length) {
```

Comment:
Is this the convention that the RN source uses to check for `nil` and a string not being empty?